src/cryptoadvance/specter/static/helper-modules/formatting.js

```diff
@@ -0,0 +1,13 @@
+function capitalize(str){
+	return str.charAt(0).toUpperCase()+str.substring(1);
+}
+
+function numberWithCommas(x) {
```

Comment:
In  https://github.com/cryptoadvance/specter-desktop/pull/1901 , I should replace the remaining occurrences of `numberWithCommas` , such that we can remove this function.

Reply:
OK, even better :-).